packages/forms/src/directives/reactive_directives/form_control_name.ts

```diff
@@ -238,7 +242,9 @@ export class FormControlName extends NgControl implements OnChanges, OnDestroy {
    * Returns an array that represents the path from the top-level form to this control.
    * Each index is the string name of the control on that level.
    */
-  get path(): string[] { return controlPath(this.name, this._parent !); }
+  get path(): string[] {
+    return controlPath(this.name == null ? null : this.name.toString(), this._parent !);
```

Comment:
I think this is what we need to do to make it fully backwards compatible.

```suggestion
    return controlPath(this.name == null ? this.name : this.name.toString(), this._parent !);
```

Reply:
Done 👍